event-stream-processing/src/shared/regex.service.IIS.spec.ts

```diff
@@ -0,0 +1,71 @@
+import {RegexService} from './regex.service';
+import {LoggerService} from '../util/logger.service';
+import {OsDocument} from '../types/os-document';
+
+const WIN_IIS_ACCESS_LOG_EVENT_SIGNATURE = Object.freeze({
+  event: Object.freeze({
+    'kind': 'event',
+    'category': 'web',
+    'dataset': 'iis.access',
+    '@metadata': {
+      iisAccessLog: true,
+    },
+  }),
+});
+
+/* eslint-disable max-len,camelcase,@typescript-eslint/no-unsafe-call */
+const regex01=/^(?<extract_timestamp>\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2})\s((\S+)\s){2}(?<source__ip>(\S+))\s(?<http__request__method>(\S+))\s(?<url__full>(\S+))\s(\S+)\s(?<url__port>(\S+))\s(-|\S+)\s(?<client__ip>(-|\S+))\s(?<extract_httpVersion>(\S+))\s(?<user_agent__original>(\S+))\s((\S+)\s){2}(?<server__address>(\S+))\s(?<http__response__status_code>(-|\d+))\s((-|\d+)\s){2}(?<http__request__bytes>(-|\d+))\s(?<http__response__bytes>(-|\d+))\s(?<event__duration>(-|\d+)).?$/;
+/* eslint-enable max-len */
+
+const regexArr = [regex01];
+
+describe('RegexService', () => {
```

Comment:
Are any of these unique tests of the regex service? If there are any it should be merged into regex.service.sec.ts. Otherwise, this file can be dropped.

Reply:
Merged tests into regex.service.spec.ts